arm/applications/application.cpp

```diff
@@ -44,15 +45,18 @@ hal::status application(sjsu::arm::application_framework& p_framework)
   hal::delay(clock, 1000ms);
 
   while (true) {
-    
-    auto timeout = hal::create_timeout(clock, 5s);
-    commands = mission_control.get_command(timeout).value();
+    if(loop_count==10) {
+      auto timeout = hal::create_timeout(clock, 5s);
```

Comment:
```suggestion
      auto timeout = hal::create_timeout(clock, 100ms);
```

You do not want to deadlock the rover for 5s while you figure out whether or not you are connected. This time is too long and if the rover is no-responsive could harm someone. We do have the E-stop but this time should be reduced. In this configuration, you'll reach this every 60ms (good choice, I think my best with ExpressJS was 50ms or so), then an additional 100ms. So every reconnect attempt happens around every 160ms. Once we get this pass in, we'll need to go over how to improve how timing is done for this control loop. It makes more sense to push the frequency of checks INTO the mission control driver itself rather than having to manage it outside. But this was my suggestion as a quick fix.